packages/opentelemetry-cloud-trace-exporter/test/exporter.test.ts

```diff
@@ -174,15 +165,14 @@ describe('Google Cloud Trace Exporter', () => {
           return def;
         }
       );
-
       debug = sinon.spy();
       info = sinon.spy();
       warn = sinon.spy();
       error = sinon.spy();
-      sinon.replace(logger, 'debug', debug);
-      sinon.replace(logger, 'info', info);
-      sinon.replace(logger, 'warn', warn);
-      sinon.replace(logger, 'error', error);
+      sinon.replace(diag, 'debug', debug);
+      sinon.replace(diag, 'info', info);
+      sinon.replace(diag, 'warn', warn);
+      sinon.replace(diag, 'error', error);
```

Comment:
Same as above, can we delete these variables altogether?

Reply:
The `error` is one is being used on line 290 I have left that one in but removed the others